components/Header.js

```diff
@@ -1,38 +1,35 @@
 import styled from 'styled-components';
-import Image from 'next/image';
 import Link from 'next/link';
+import useLocalStorage from '../hooks/useLocalStorage';
 
 export default function Header() {
   return (
     <HeaderWrapper>
-      <HeadLine>DAILY BREAD</HeadLine>
-      <Link href={'/shoppingcart'}>
+      <Link href={'/'}>
         <Anchor>
-          <Image
-            alt="shopping cart icon"
-            src="https://res.cloudinary.com/drthrskm2/image/upload/v1665657772/capstone-project/icons/shoppingbasket_aubvbk.svg"
-            layout="fixed"
-            width={40}
-            height={40}
-          />
+          <HeadLine>DAILY BREAD</HeadLine>
         </Anchor>
       </Link>
     </HeaderWrapper>
   );
 }
 
 const HeaderWrapper = styled.header`
+  background-color: #ffffff;
   display: flex;
   justify-content: center;
   width: 100%;
+  height: 3.899rem;
```

Comment:
or will the 0.001rem break your app?
```suggestion
  height: 3.9rem;
```